examples/resources/plugins/teleport-jira.toml

```diff
@@ -1,21 +1,21 @@
 # example jira plugin configuration TOML file
 [teleport]
-auth-server = "example.com:3025"  # Auth GRPC API address
-client-key = "/var/lib/teleport/plugins/jira/auth.key" # Teleport GRPC client secret key
-client-crt = "/var/lib/teleport/plugins/jira/auth.crt" # Teleport GRPC client certificate
-root-cas = "/var/lib/teleport/plugins/jira/auth.cas"   # Teleport cluster CA certs
+auth_server = "example.com:3025"                       # Teleport Auth Server GRPC API address
+client_key = "/var/lib/teleport/plugins/jira/auth.key" # Teleport GRPC client secret key
+client_crt = "/var/lib/teleport/plugins/jira/auth.crt" # Teleport GRPC client certificate
+root_cas = "/var/lib/teleport/plugins/jira/auth.cas"   # Teleport cluster CA certs
 
 [jira]
 url = "https://example.com/jira"    # JIRA URL. For JIRA Cloud, https://[my-jira].atlassian.net
 username = "bot@example.com"        # JIRA username
-api-token = "token"                 # JIRA API Basic Auth token
+api_token = "token"                 # JIRA API Basic Auth token
 project = "MYPROJ"                  # JIRA Project key
 
 [http]
-listen = ":8081"          # JIRA webhook listener
-# host = "example.com"    # Host name by which bot is accessible
-# https-key-file = "/var/lib/teleport/plugins/jira/server.key"  # TLS private key
-# https-cert-file = "/var/lib/teleport/plugins/jira/server.crt" # TLS certificate
+# listen_addr = ":8081" # Network address in format [addr]:port on which webhook server listens, e.g. 0.0.0.0:443
+# public_addr = "example.com" # URL on which webhook server is accessible externally, e.g. [https://]teleport-jira.example.com
```

Comment:
Address at the end should match the address at the beginning

i.e. `public_addr = "teleport-jira.example.com"`